pkgs/development/libraries/folly/default.nix

```diff
@@ -23,8 +42,16 @@ stdenv.mkDerivation rec {
     libevent
     libiberty
     openssl
+    lz4
+    lzma
+    zlib
+    jemalloc
+    libunwind
+    fmt
   ];
 
+  cmakeFlags = [ "-DBUILD_SHARED_LIBS=ON" ];
+
   enableParallelBuilding = true;
```

Comment:
Cmake set this automatically. This should be removed.

Reply:
Really? Here's a comparison of the build products with and without that line:
```
@@ -778,9 +778,9 @@
     │       ├── folly-config.cmake
     │       ├── folly-targets.cmake
     │       └── folly-targets-release.cmake
-    ├── libfolly.a
-    ├── libfollybenchmark.a
-    ├── libfolly_test_util.a
+    ├── libfollybenchmark.so
+    ├── libfolly.so
+    ├── libfolly_test_util.so
     └── pkgconfig
         └── libfolly.pc

```

Reply:
Ah, you meant the parallel building - I got confused by the way Github displays the comment :slightly_smiling_face: